### PR TITLE
CDN manager config

### PR DESCRIPTION
**Pull request by 244372610**
Signed-off-by: 孙伟鹏 <weipeng.swp@alibaba-inc.com>

### CODE REVIEW

File: charts/dragonfly/templates/cdn/cdn-configmap.yaml [hunk 1]
```diff
@@ -44,6 +44,14 @@ data:
...
     jaeger: http://jaeger-all-in-one-collector.{{ $.Release.Namespace }}.svc.{{ $.Values.clusterDomain }}:14268/api/traces
     {{- end }}
+    manager:
+      { { - if .Values.manager.enable } }
+      addr: { { template "dragonfly.manager.fullname" . } }.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:{{ .Values.manager.grpcPort }}
+      {{- else }}
+      addr: { { .Values.externalManager.host } }:{{ .Values.externalManager.grpcPort }}
+      {{- end }}
+      cdnClusterID: { { .Values.cdn.config.base.manager.cdnClusterID } }
+      keepAlive:
```
Comment (yxxhero) [inline charts/dragonfly/templates/cdn/cdn-configmap.yaml:54]: keepAlive no value?

Comment (gaius-qi) [reply charts/dragonfly/templates/cdn/cdn-configmap.yaml:54]: @244372610

**Review by gaius-qi — CHANGES_REQUESTED** [review]

File: charts/dragonfly/templates/cdn/cdn-configmap.yaml [hunk 2]
```diff
@@ -44,6 +44,14 @@ data:
...
     jaeger: http://jaeger-all-in-one-collector.{{ $.Release.Namespace }}.svc.{{ $.Values.clusterDomain }}:14268/api/traces
     {{- end }}
+    manager:
+      { { - if .Values.manager.enable } }
+      addr: { { template "dragonfly.manager.fullname" . } }.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:{{ .Values.manager.grpcPort }}
+      {{- else }}
+      addr: { { .Values.externalManager.host } }:{{ .Values.externalManager.grpcPort }}
+      {{- end }}
+      cdnClusterID: { { .Values.cdn.config.base.manager.cdnClusterID } }
+      keepAlive: { { .Values.cdn.config.base.manager.keepAlive.interval } }
```
Comment (gaius-qi) [inline charts/dragonfly/templates/cdn/cdn-configmap.yaml:54]: https://github.com/dragonflyoss/helm-charts/blob/main/charts/dragonfly/templates/scheduler/scheduler-configmap.yaml#L34

**Review by gaius-qi — CHANGES_REQUESTED** [review]

**Review by gaius-qi — APPROVED** [review]
LGTM

**Pull request merged**